README.md

```diff
@@ -17,6 +17,10 @@ Currently planning to use:
 - backend: Supabase
 - hosting: Vercel
 
+## Project Board - Road Map
```

Comment:
Road Map -> roadmap

Project Board -> Project board

Reply:
Got it :D